.travis.yml

```diff
@@ -88,6 +88,8 @@ install:
   - travis_retry sudo luarocks install busted
   # Install luacheck for "make check-qa".
   - if [ "$DO_CHECKQA" = 1 ]; then travis_retry sudo luarocks install luacheck; fi
+  # Install depgraph for "make check-requires".
+  - travis_retry sudo luarocks install depgraph
```

Comment:
Please add a `CHECK_DEPGRAPH` variable and only instal/use depgraph if it's set. Only 1 target should run this. Until Docker is working, those luarocks are "costly".